CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 -Features for next release
 
+### Breaking Changes
+- **Amazon Cognito**
+  - **Breaking Change** Custom auth will set the right username after a signin flow. Earlier if you use an alias (ex email) for custom auth signin, after successfull signIn `AWSMobileClient.default.username` would return the entered alias. This has been changed in this release and instead of the returning the alias, `AWSMobileClient.default.username` will return the actual Cognito username of the user. (See [Issue #3194](https://github.com/aws-amplify/aws-sdk-ios/issues/3194), [PR #3198](https://github.com/aws-amplify/aws-sdk-ios/pull/3198))
```

Comment:
```suggestion
  - **Breaking Change** Custom auth now sets the correct `username` after a sign in flow. Before, if a user signed in with an alias (for example, an email) in a custom auth sign in flow, after a successful signIn, `AWSMobileClient.default().username` would return that entered alias. This has been fixed in this release to conform to the behavior in the standard SRP flow: instead of the returning the alias, `AWSMobileClient.default().username` will return the actual Cognito username of the user. (See [Issue #3194](https://github.com/aws-amplify/aws-sdk-ios/issues/3194), [PR #3198](https://github.com/aws-amplify/aws-sdk-ios/pull/3198))
```